### ISSUE TITLE
Export plan as image

### ISSUE THREAD

**Issue by RemiDesgrange** [AlexTatiyants/pev#35]
Hi,

First, thanks for this amazing piece of software, it helps me a lot.

What do you think about exporting the generated plan graph as an image (png, svg, pdf, whatever...) ? do you plan something like that ?

**Comment by gsusrafael** [comment]
I agree with @RemiDesgrange, this could be useful when you are sharing and discussing about optimizations related to a query among a team (devs,sysadmins,dba, even devops)

**Comment by felixfbecker** [comment]
I'd love to be able to export as an SVG

**Comment by ChampionNan** [comment]
Is there any support for this?
